.github/workflows/get-next-meeting-date.yml

```
@@ -0,0 +1,60 @@
+name: Get Next Meeting Date
+on:
+  push:
+    branches: ["scrape-next-meeting-date"] # Runs on pushes to the scrape-next-meeting-date branch
+
+  schedule:
+    - cron: "0 0 * * 0" # Runs at midnight every Sunday
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets the GITHUB_TOKEN permissions to allow committing changes, so we can update the next meeting date
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  execute-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: "hailexiao/boston-liquor-license-tracker"
+          ref: "scrape-next-meeting-date"
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: "npm"
+          cache-dependency-path: client/package-lock.json
+
+      - name: Install axios
+        working-directory: ./client
+        run: npm i axios
+
+      - name: Install cheerio
+        working-directory: ./client
+        run: npm i cheerio
+
+      - name: Install fs
+        working-directory: ./client
+        run: npm i fs
+
+      - name: Compile next meeting date script
+        working-directory: ./client/src/scripts
+        run: npx tsc -t es2022 -m nodenext --moduleResolution nodenext next-meeting-date.ts
+
+      - name: Run next meeting date script
+        working-directory: ./client/src/scripts
+        run: node next-meeting-date.js
+
+      - name: Commit changes to repository
+        run: |
+          git config --local user.name "GitHub Actions"
+          git config --local user.email "hello@codeforboston.org"
+          git remote add upstream https://github.com/hailexiao/boston-liquor-license-tracker.git
+          git commit -am "Update next meeting date"
```

🌴 🌴 🌴 Non-blocking comment: Adding some kind of tag in the commit message will make it easier to see these commits while browsing the git log. Doesn't necessarily need to be in this structure or this text. ^^

```suggestion
          git commit -am "[SCRIPT] Update next meeting date"
```